packages/icons/src/pause-unfilled.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please use this:

```
<svg viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
	<path d="M27 16C27 9.92487 22.0751 5 16 5C9.92487 5 5 9.92487 5 16C5 22.0751 9.92487 27 16 27C22.0751 27 27 22.0751 27 16ZM29 16C29 23.1797 23.1797 29 16 29C8.8203 29 3 23.1797 3 16C3 8.8203 8.8203 3 16 3C23.1797 3 29 8.8203 29 16ZM12.5 11.9998C12.5 11.4476 12.9477 10.9998 13.5 10.9998C14.0522 10.9998 14.5 11.4476 14.5 11.9998V19.9998C14.5 20.5521 14.0522 20.9998 13.5 20.9998C12.9477 20.9998 12.5 20.5521 12.5 19.9998V11.9998ZM17.4999 11.9998C17.4999 11.4476 17.9476 10.9998 18.4999 10.9998C19.0522 10.9998 19.4999 11.4476 19.4999 11.9998V19.9998C19.4999 20.5521 19.0522 20.9998 18.4999 20.9998C17.9476 20.9998 17.4999 20.5521 17.4999 19.9998V11.9998Z" />
</svg>
```

Reply:
Thanks @rafaelblink This was able to solve the issue of the icon being not visible in storybook